main/streams/plain_wrapper.c

```diff
@@ -918,7 +918,8 @@ static int php_stdiop_set_option(php_stream *stream, int option, int value, void
 				case PHP_STREAM_SYNC_FDSYNC:
 					return php_stdiop_sync(stream, 1) == 0 ? PHP_STREAM_OPTION_RETURN_OK : PHP_STREAM_OPTION_RETURN_ERR;
 			}
-
+			/* TODO Should this fallthrough? */
+			ZEND_FALLTHROUGH;
```

Comment:
I think we should return `PHP_STREAM_OPTION_RETURN_NOTIMPL` here...

Reply:
This might indeed be correct, I was just surprised that CI was failing because of this, so I made a quick commit adding this to see if there are any other ones.

Seems this is the last one which is also finding an actual bug :-)

Reply:
Or maybe even `PHP_STREAM_OPTION_RETURN_ERR`, as this means an incorrect option value was passed.